packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -752,6 +778,59 @@ export default async function initSanity(
   function getOrganizationsWithAttachGrant(organizations: ProjectOrganization[]) {
     return pFilter(organizations, (org) => hasProjectAttachGrant(org.id), {concurrency: 3})
   }
+
+  async function createOrAppendEnvVars(filename: string) {
+    const variables = {
+      SANITY_PROJECT_ID: projectId,
+      SANITY_DATASET: datasetName,
+    }
+
+    try {
+      // use vercel's framework detector
+      const framework: Framework | null = await detectFrameworkRecord({
+        fs: new LocalFileSystemDetector(workDir),
+        frameworkList: frameworks as readonly Framework[],
+      })
+
+      if (framework && framework.slug !== 'sanity') {
+        print(
+          `\nDetected framework ${chalk.blue(framework?.name)}, using prefix '${
+            framework.envPrefix
+          }'`
```

Comment:
What happens if it detects a framework that doesn't have a prefix, like `remix`?
Does it print:
```
Detected framework Remix, using prefix ''
```
?

Reply:
I forgot to check the truthiness of prefix. Good catch!